client/src/Views/Home/Prospects/Prospects.js

```diff
@@ -11,151 +18,305 @@ import ProspectTableCheckbox from "Components/Checkbox/ProspectTableCheckbox";
+                    spacing={2}
+                    className={this.props.classes.list_entire}
+                  >
+                    <React.Fragment>
+                      <Table size="small" className={this.props.classes.table}>
+                        <TableHead className={this.props.classes.table_header}>
+                          <TableRow className={this.props.classes.table_header}>
+                            <TableCell
+                              className={this.props.classes.table_header_cell}
+                              align="center"
+                            >
+                              <ProspectTableCheckbox
+                                disableRipple={true}
+                                checked={this.state.allChecked}
+                                onChange={e => this.handleChange(e, "all")}
+                                className={
+                                  this.props.classes.table_checkbox_checked
+                                }
+                              />
+                            </TableCell>
+                            <TableCell
+                              className={this.props.classes.table_header_cell}
+                            >
+                              Email
+                            </TableCell>
+                            <TableCell
+                              className={this.props.classes.table_header_cell}
+                              align="center"
+                            >
+                              <CloudIcon
+                                className={this.props.classes.cloud_icon}
+                              ></CloudIcon>
+                            </TableCell>
+                            <TableCell
+                              className={this.props.classes.table_header_cell}
+                              align="center"
+                            >
+                              Status
+                            </TableCell>
+                            <TableCell
+                              className={this.props.classes.table_header_cell}
+                              align="center"
+                            >
+                              Owner
+                            </TableCell>
+                            <TableCell
+                              className={this.props.classes.table_header_cell}
+                              align="center"
+                            >
+                              Last Contacted
+                            </TableCell>
+                            <TableCell
+                              className={this.props.classes.table_header_cell}
+                              align="right"
+                            >
+                              Emails
+                            </TableCell>
+                          </TableRow>
+                        </TableHead>
+                        <TableBody>{filteredProspectList}</TableBody>
```

Comment:
There is a lot of repeated code in this prospects table, could use a refactor, but I understand we were prepping for the presentation